client/actions/types.js

```diff
@@ -17,3 +17,7 @@ export const DAYS_FETCH_DATA_SUCCESS = 'DAYS_FETCH_DATA_SUCCESS';
 export const DAILY_RESULT_HAS_ERRORED = 'DAILY_RESULT_HAS_ERRORED';
 export const DAILY_RESULT_IS_LOADING = 'DAILY_RESULT_IS_LOADING';
 export const DAILY_RESULT_FETCH_DATA_SUCCESS = 'DAILY_RESULT_FETCH_DATA_SUCCESS';
+
+export const SIMULATIONS_HAS_ERRORED = 'SIMULATIONS_HAS_ERRORED';
+export const SIMULATIONS_IS_LOADING = 'SIMULATIONS_IS_LOADING';
+export const SIMULATIONS_FETCH_DATA_SUCCESS = 'SIMULATIONS_FETCH_DATA_SUCCESS';
```

Comment:
It doesn't seem optimal to copy paste those three lines every time a new model is added. Maybe we should do something like this: function isLoading(subject){ return subject.toUpperCase() + "_IS_LOADING" }